docs/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Pytentiostat'
```

Comment:
These don't look quite right....what are they for?    If we want to define authors and copyright for the project we should use the python builtin variables ``__author__`` etc.., but it is generally better to keep a complete list of authors and contributors in the top level of the package, and perhaps define these in the ``__init__.py`` file so they are available in the namespace of the whole project.

Reply:
These are for the html footer. They were automatically created when I was doing the quickstart build function from sphinx. This list will be on the documentation, not intended to replace the top level list.